Mage.Tests/src/test/java/org/mage/test/cards/abilities/oneshot/destroy/UnlicensedDisintegrationTest.java

```diff
@@ -0,0 +1,115 @@
+package org.mage.test.cards.abilities.oneshot.destroy;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+
+/*
+Unlicensed Disintigration - Hi! Noticed that everytime that i succesfully cast 
+Unlicensed Disintigration with an artifact on the board the opponent wont lose 3 life. 
+The creature dies but the last piece of text does not work (teM, 2020-02-24 15:17:36)
+*/
+public class UnlicensedDisintegrationTest extends CardTestPlayerBase{
+    
+    @Test
+    public void testDestroyCreatureLifeLoss(){
+        addCard(Zone.BATTLEFIELD, playerA, "Mountain");
+        addCard(Zone.BATTLEFIELD, playerA, "Swamp");
+        addCard(Zone.BATTLEFIELD, playerA, "Swamp");
```

Comment:
Add card's amount at the end, you don't need to add it as same code call:
`addCard(Zone.BATTLEFIELD, playerA, "Swamp", 2);`